### PR TITLE
Add details for how to setup and build the documentation to the readme

### PR DESCRIPTION
**Pull request by settermjd**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Because I'm not aware that this is documented anywhere, I thought that it might be helpful to add it here, just in case others need to know how to prepare their local development environment, to build the documentation, and how to view generated documentation, locally. I hope that it's of help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
-changes and milestones for the Laminas Project and related subproject.
+changes and milestones for the Laminas Project and related subproject, and to show how to setup and update the documentation locally.
+
+## Setup
+
+To prepare your local development environment to build the documentation locally, follow the steps below:
+
+1. Install [Python 3](https://www.python.org/downloads/) and [PIP](https://pip.pypa.io/en/stable/installing/)
+2. Install the [Python PyMdown extension](https://facelessuser.github.io/pymdown-extensions/installation/#requirements)
+3. [Install MkDocs](https://www.mkdocs.org/#installation) and all of its dependencies
```
Comment (froschdesign) [inline README.md:12]: A package manager should be used for the installation of Python and this is already included on mkdocs.org.

```suggestion
1. [Install MkDocs](https://www.mkdocs.org/#installation) and all of its dependencies (The installation description for MkDocs also includes the installation of Python.)
2. Install the [Python PyMdown extension](https://facelessuser.github.io/pymdown-extensions/installation/#installation_1)
```

**Comment by settermjd** [comment]
Thanks, @froschdesign

**Comment by froschdesign** [comment]
@settermjd 
I think this should be sufficient.

**Comment by froschdesign** [comment]
@settermjd 
Btw. please of a look at the signed of: https://github.com/laminas/documentation/pull/20/checks?check_run_id=1533317377

**Comment by settermjd** [comment]
> @settermjd
> Btw. please of a look at the signed of: https://github.com/laminas/documentation/pull/20/checks?check_run_id=1533317377

Just working on that at the moment.

**Comment by settermjd** [comment]
Done!

**Pull request merged**